tailwind.config.js

```diff
@@ -1,6 +1,15 @@
 module.exports = {
     theme: {
-        extend: {}
+        fontFamily: {
+            raleway: ['Raleway'],
+            robotoMono: ['Roboto Mono']
+        },
+        extend: {
+            colors: {
+                'light-gray': '#F1F1F1',
+                'filter-blue': 'rgba(81, 188, 243, 0.53)'
```

Comment:
Try following the color naming convention in the framework. For example, `blue-100`

Reply:
I resolved to naming it blue-1100 to avoid overwriting tailwind's default variable with that color